javascript_nodejs/50.contoso-cafe-bot/dialogs/chitChat/index.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Help intent name from ../../mainDialog/resources/cafeDispatchModel.lu 
+const CHITCHAT_INTENT_NAME = 'ChitChat';
+
+// Help, ChitChat and QnA share the same QnA Maker model. So, just export the Help intent name here. 
+// This is used by MainDialog to dispatch to the appropriate child dialog.
+// The name needs to match the intent name returned by LUIS. 
+module.exports = {
+    Name : CHITCHAT_INTENT_NAME
+};
```

Comment:
this is perhaps pedantic, but I think it would make sense to define a dialog class for each QnA model.

export class ChitChatDialog extends QnaMakerDialog {
    constructor() {
            super(CHITCHAT_DIALOG_ID)
    }
}

This would make dispath always dispatch to a dialog, in this case a QnAMakerDialog for chit chat, and it sets up chit chat to decide to be multi-turn in the future.  For example, knock knock joke is a multi-turn chitchat